doc/dev-reference/getting_started.rst

```diff
@@ -32,7 +32,7 @@ Very Quick Start
    Generators that might work include:
       * ``Xcode`` --- OS X Xcode projects.
```

Comment:
Xcode support has not been verified to work, and is not a supported, I would probably remove it or add a mention that it's an unsupported configuration.

Reply:
Noted it is unsupported.